app/controllers/DraftMetadataUploadController.scala

```diff
@@ -35,4 +42,30 @@ class DraftMetadataUploadController @Inject() (
       }
     }
   }
+
+  def saveDraftMetadata(consignmentId: java.util.UUID): Action[MultipartFormData[TemporaryFile]] = secureAction.async(parse.multipartFormData) {
+
+    implicit request: Request[MultipartFormData[TemporaryFile]] =>
+      val successPage = routes.DraftMetadataChecksController.draftMetadataChecksPage(consignmentId)
+      val token = request.asInstanceOf[Request[AnyContent]].token
+
+      val uploadBucket = s"${applicationConfig.draft_metadata_s3_bucket_name}-${applicationConfig.frontEndInfo.stage}"
+
+      val uploadKey = s"$consignmentId/draft-metadata.csv"
+
+      def uploadMetaData: IO[Result] = for {
+        firstFilePart <- fromOption(request.body.files.headOption)(new RuntimeException("No meta data file provided"))
+        file <- fromOption(request.body.file(firstFilePart.key))(new RuntimeException("No meta data file provided"))
+        draftMetadata <- fromOption(Using(scala.io.Source.fromFile(file.ref.getAbsoluteFile))(_.mkString).toOption)(new RuntimeException("No meta data file provided"))
+        _ <- uploadService.uploadDraftMetadata(uploadBucket, uploadKey, draftMetadata)
+        _ <- IO.fromFuture(IO { draftMetadataService.triggerDraftMetadataValidator(consignmentId, token.toString) })
```

Comment:
Need to get the bearer access token to pass into the method: ... `draftMetadataService.triggerDraftMetadataValidator(consignmentId, token.bearerAccessToken.getValue) ...`